synfig-studio/src/gui/modules/mod_palette/dock_paledit.cpp

```diff
@@ -371,6 +379,16 @@ Dock_PalEdit::erase_color(int i)
 	refresh();
 }
 
+void
+Dock_PalEdit::invert_color(int i)
+{
+	palette_[i].color.set_r(1.00f-palette_[i].color.get_r());
+	palette_[i].color.set_g(1.00f-palette_[i].color.get_g());
+	palette_[i].color.set_b(1.00f-palette_[i].color.get_b());
```

Comment:
There is an `operator ~` in `synfig::Color` that invert color :)

I don't know if this (interesting) PR will be accepted, but I think maybe it would be good to use this operator.

Reply:
Thank you very much for the suggestion! I will implement the inverting using that ~ operator and commit it to the branch.